translation/en/RedirectedResources/assets/abdata/adv/scenario/c20/20/1000/translation.txt

```diff
@@ -1,16 +1,16 @@
-//「すみませーん！　少しお時間よろしいでしょうか？」=
-//「あっ、はい。何ですか？」=
-//「ありがとうございます！　私、この近くの結婚式場に勤めているウェディングプランナーの玉城と申します！」=
-//「う、ウェディングプランナー？　どうして、そんな人が俺たちに声を……？」=
-//「現在、私どもの式場にてブライダルフェアを行っておりまして、模擬挙式のモデルをお願いできる方を探していたのですが……」=
-//「もしかして、俺たちにそのモデルを？」=
-//「はい！　すごくお似合いで素敵なカップルさんでしたので、是非にと声をかけさせていただきました！　いかがでしょうか？」=
-//「どうする？　俺は引き受けてみたいって思うんだけど」=
-//「興味はあります……けど、私に出来るでしょうか……？」=
-//「大丈夫だよ、あくまで模擬なんだから。それに俺も一緒だし、何かあればフォローするよ」=
-//「でしたら……引き受けて、やってみたいですっ」=
-//「オッケー。決まりだね」=
-//「お待たせしました。俺達でよければ、やらせてください」=
-//「ありがとうございます！　それでは、さっそく式場までご案内いたしますね！」=
+「すみませーん！　少しお時間よろしいでしょうか？」= Excuse me, do you have a moment?
+「あっ、はい。何ですか？」=Yeah?
+「ありがとうございます！　私、この近くの結婚式場に勤めているウェディングプランナーの玉城と申します！」=Thank you, sorry to distrub you. My name is Tamashiro, and I'm a wedding planner who works for a venue nearby.
+「う、ウェディングプランナー？　どうして、そんな人が俺たちに声を……？」=A wedding planner? Why did you want to speak with us?
+「現在、私どもの式場にてブライダルフェアを行っておりまして、模擬挙式のモデルをお願いできる方を探していたのですが……」=We are currently holding a bridal fair in our ceremony hall, and I'm looking for a someone, a couple of people to exact, who could for a model for a simulated wedding ceremony.
+「もしかして、俺たちにそのモデルを？」=And you think we could be that model?
+「はい！　すごくお似合いで素敵なカップルさんでしたので、是非にと声をかけさせていただきました！　いかがでしょうか？」=I do. You just seemed like such a cute couple, I had to ask. So, what do you say?
+「どうする？　俺は引き受けてみたいって思うんだけど」=What do you think? I think it sounds interesting.
+「興味はあります……けど、私に出来るでしょうか……？」=I'm do too... but I'm not sure I can do it.
+「大丈夫だよ、あくまで模擬なんだから。それに俺も一緒だし、何かあればフォローするよ」=You'll be totally fine, there's nothing to worry about, it's not like it's a real wedding. Plus, I'll be there if you need help.
+「でしたら……引き受けて、やってみたいですっ」=Then... I'll do it!
+「オッケー。決まりだね」=Ok, sounds good.
+「お待たせしました。俺達でよければ、やらせてください」=Ma'am, it sounds good to us. Let's do it.
```

Comment:
This is more like
"Sorry for making you wait. If you're fine with us, then please let us do it."